### ISSUE TITLE
"Diff assembly in a graph” option seems to be missing

### ISSUE THREAD

**Issue by Sherlock88** [joxeankoret/diaphora#58]
I created two tiny C programs with only one function apart from main(). The two versions differ by one line of source code. Diaphora correctly detects the mismatch and displays the modified function under "_Unmatched in Secondary_" tab. However, if I click on the function in the listing, I am unable to find an option called “_Diff assembly in a graph_” as your blog example illustrates.  Surprisingly, the same function appears under both "_Unmatched in Primary_" and "_Unmatched in Secondary_" tabs.

**Comment by joxeankoret** [comment]
It looks odd. Can you share the small programs, please?

**Comment by Sherlock88** [comment]
**test1.c**

```
#include <stdio.h>

int f(int a, int b)
{
	return a + b;
}

int main()
{
	printf("%d\n", f(4, 5));
	return 0;
}
```

**test2.c**

```
#include <stdio.h>

int f(int a, int b)
{
	int  c =(a + b) * (a - b);
	return c;
}

int main()
{
	printf("%d\n", f(4, 5));
	return 0;
}
```

**Comment by joxeankoret** [comment]
I think I know. Which compiler?

**Comment by Sherlock88** [comment]
Compiled with `gcc version 4.8.4 (Ubuntu 4.8.4-2ubuntu1~14.04.1) ` on `Ubuntu 14.04.2, 64 bit`

**Comment by joxeankoret** [comment]
I can reproduce. It seems the problem is that the function name has just a single letter. I will fix it in a while.

**Comment by Sherlock88** [comment]
I see. Let me try with reasonably longer names in that case.

**Comment by joxeankoret** [comment]
No, it wasn't that. It was a bug in Diaphora. When comparing databases with the actual same call graph, as a (wrong) speed up, it ignored function names. Bug fixed with commit c17e7c3550864254b80e4141f97d750cfa125a09.

Thanks for reporting!
